x-pack/plugins/ui_actions_enhanced/public/can_inherit_time_range.test.ts

```diff
@@ -6,7 +6,7 @@
 import { canInheritTimeRange } from './can_inherit_time_range';
 /** eslint-disable */
 import { HelloWorldContainer } from '../../../../src/plugins/embeddable/public/lib/test_samples';
-import { HelloWorldEmbeddable } from '../../../../examples/embeddable_examples/public';
+import { HelloWorldEmbeddable } from '../../../../examples/embeddable_examples/public/hello_world';
```

Comment:
I understand this is required for tests. But the whole change still doesn't enforce `Prefer one way of doing things`.